gatsby-node.js

```diff
@@ -5,24 +5,40 @@
  */
 
 // You can delete this file if you're not using it
-const path = require(`path`)
+const path = require(`path`);
 
 exports.createPages = ({ graphql, actions }) => {
-  const { createPage } = actions
-  const page = path.resolve(`src/pages/index.js`)
+  const { createPage } = actions;
+  const page = path.resolve(`src/pages/index.js`);
   createPage({
     path: `/webpack`,
     component: page,
     context: {
       selectedTab: 'webpack',
     },
-  })
+  });
 
   createPage({
     path: `/parcel`,
     component: page,
     context: {
       selectedTab: 'parcel',
     },
-  })
-}
+  });
+
+  createPage({
+    path: `/rollup`,
+    component: page,
+    context: {
+      selectedTab: 'rollup',
+    },
+  });
+};
+
+exports.onCreateWebpackConfig = ({ actions }) => {
+  actions.setWebpackConfig({
```

Comment:
Ah, as I mentioned, I was facing some error while using prettier api to format the code snippet. So this piece of code should be added here so that it works.  I will keep all these comments and update the PR once my local systems issue is resolved.

Reply:
ok I see! a comment about this in the code will be very valuable once you get your systems up and running again

Reply:
Sure! My system issue is happening only with last commit. If I reset hard to previous commit and start the server, it is working fine. I tested the same branch with latest commit in my friend's mac, it is working fine! this is weird! I am setting up a ubuntu virutal machine to see if it helps.